src/components/audio-call-game/game/startNewGame/game.ts

```diff
@@ -70,7 +73,12 @@ export async function startNewGame(event: Event | null, startPage: HTMLElement |
         updateState('indexWord', 0);
         let listWords;
         if (statusAuth) {
-          listWords = +currentLevel !== USER_LEVEL ? await getAuthWords(+currentLevel, randomPage) : await getAllUserWordsWithData();
+          if (+currentLevel === USER_LEVEL) {
+            listWords = await getAllUserWordsWithData();
+          } else {
+            listWords = await getAggregatedWords(undefined, WORD_AMOUNT_FOR_PAGE, +currentLevel, randomPage);
```

Comment:
Consider naming
`listWords` to `aggregatedWords` (so it'll be less confusing)

Reply:
corrected, [8a60838](https://github.com/Jeneko/rslang/pull/111/commits/8a60838f21f111f3851ba6b3f8d62598dc53ece6)